### ISSUE TITLE
Should not run npm ci after manually installing a dependency.

### ISSUE THREAD

**Issue by dg-eparizzi** [tkglaser/pkgmon#1]
After properly setting pkgmon as described, if I manually install a new dependency "npm install some-package", after I run my start command, pkgmon will detect a change in my package.json and run the full npm ci which takes a lot of time.

There has to be a better way to deal with this.

Have you thought about using npm prune and npm install instead of npm ci?

**Comment by tkglaser** [comment]
Good point. I'll have some experiments with `prune/install` instead of `ci`. Or maybe even add an option for it?
What you can do in the mean time is to run a `pkgmon reset` after you installed a package locally. That'll avoid the extra `npm ci` run on your box.
